upgrading/rosa-upgrading-sts.adoc

```diff
@@ -14,16 +14,16 @@ To plan an upgrade, review the xref:../rosa_architecture/rosa_policy_service_def
 Upgrades are manually initiated or automatically scheduled. Red Hat Site Reliability Engineers (SREs) monitor upgrade progress and remedy any issues encountered.
 
 [id="rosa-sts-upgrading-a-cluster-with-sts"]
-== Upgrading a ROSA cluster that uses STS
+== Upgrading a ROSA Classic cluster
 
-There are two methods to upgrade {product-title} (ROSA) clusters that uses the AWS Security Token Service (STS):
+There are two methods to upgrade Classic {product-title} (ROSA) clusters:
 
 * Individual upgrades through the ROSA CLI (`rosa`)
 * Individual upgrades through the {cluster-manager} console
 
 [NOTE]
 ====
-For steps to upgrade a ROSA cluster that does not use the AWS Security Token Service (STS), see xref:../upgrading/rosa-upgrading.adoc#rosa-upgrading[Upgrading ROSA clusters].
+When following a scheduled upgrade policy, there might be a delay of an hour or more before the upgrade process begins, even if it is an immediate upgrade. Additionally, the duration of the upgrade might vary based on your workload configuration.
```

Comment:
```suggestion
When you follow a scheduled upgrade policy, a delay of an hour or more before the upgrade process begins might occur, even if the upgrade is configured to begin immediately. Additionally, the duration of the upgrade might vary based on your workload configuration.
```
- Consider editing to avoid "there is" and "it is" constructions, [per the ISG](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#t). Generally, those types of constructions lead to wordiness and weaken a sentence's impact by getting through the subject/verb combination without really saying anything.